docs/coffeescript.js

```diff
@@ -0,0 +1,309 @@
+                    }" onclick="openModal('${x}',${imageIndex},${i})">
+                    <img
+                        src="./images/${x}-${imageIndex}.jpg"
+                        alt="Photo of a ${x}"
+                        class="preview image" />
+                    <h3 class="price-text coffee-name">${products[i].name}</h3>
+                    <p class="coffee-description">
+                    ${products[i].description}
+                    </p>
+                    <h3 class="price-text">$${products[i].price}</h3>
+                    </div>`;
+      imageIndex++;
+    }
+  }
+  return (grid.innerHTML = gridContent);
+}
+
+coffeeSelector.addEventListener("click", (e) => {
+  e.preventDefault();
+  createMenuGrid(coffeeSelector.value);
+});
+
+teaSelector.addEventListener("click", (e) => {
+  e.preventDefault();
+  createMenuGrid(teaSelector.value);
+});
+
+dessertSelector.addEventListener("click", (e) => {
+  e.preventDefault();
+  createMenuGrid(dessertSelector.value);
+});
+
+createMenuGrid(coffeeSelector.value);
+
+// script for the  coffee-modal calculations
+function checkSizeS(value) {
+  value = Math.abs(value);
+  total = Math.abs(total);
+  const sizeS = document.getElementById("size-s");
+  total -= lastValue;
+  if (sizeS.checked == 1) {
+    total = total + value;
+  }
+  lastValue = value;
+  document.querySelector(".total h3:last-child").innerText = `$${total.toFixed(
+    2
+  )}`;
+  return total;
+}
+
+function checkSizeM(value) {
+  value = Math.abs(value);
+  total = Math.abs(total);
+  const sizeM = document.getElementById("size-m");
+  total -= lastValue;
+  if (sizeM.checked == 1) {
+    total = total + value;
+  }
+  lastValue = value;
+  document.querySelector(".total h3:last-child").innerText = `$${total.toFixed(
+    2
+  )}`;
+  return total;
+}
+
+function checkSizeL(value) {
+  value = Math.abs(value);
+  total = Math.abs(total);
+  const sizeL = document.getElementById("size-l");
+  total -= lastValue;
+  if (sizeL.checked == 1) {
+    total = total + value;
+  }
+  lastValue = value;
+  document.querySelector(".total h3:last-child").innerText = `$${total.toFixed(
+    2
+  )}`;
+  return total, lastValue;
+}
+
+function additive1(value, i) {
+  const Additive1 = document.getElementById("Additives1");
+  if (Additive1.checked == 1) {
+    total = total + value;
+  } else {
+    total = total - value;
+  }
+  document.querySelector(".total h3:last-child").innerText = `$${total.toFixed(
+    2
+  )}`;
+  return total;
+}
+
+function additive2(value, i) {
+  const Additive2 = document.getElementById("Additives2");
+  if (Additive2.checked == 1) {
+    total = total + value;
+  } else {
+    total = total - value;
+  }
+  document.querySelector(".total h3:last-child").innerText = `$${total.toFixed(
+    2
+  )}`;
+  return total;
+}
+
+function additive3(value, i) {
+  const labelAdditive3 = document.getElementById("label-Additives3");
+  const Additive3 = document.getElementById("Additives3");
```

Comment:
id and variable naming convention

Reply:
In the line 141 - Id and the variable names are changed.